docs/api.md

```diff
@@ -456,17 +465,41 @@ Emitted when a target is created, for example when a new page is opened by [`win
 
 Emitted when a target is destroyed, for example when a page is closed.
 
+#### browser.browserContexts()
+- returns: <[Array]<[BrowserContext]>>
+
+Returns all the browser contexts created in the browser. In a newly created browser, this will return
+a single instance of [BrowserContext].
+
 #### browser.close()
 - returns: <[Promise]>
 
 Closes Chromium and all of its pages (if any were opened). The [Browser] object itself is considered to be disposed and cannot be used anymore.
 
+#### browser.createIncognitoBrowserContext()
+- returns: <[Promise]<[BrowserContext]>>
+
+Creates a new incognito browser context. A typical use would be opening pages in a pristine
```

Comment:
This won't share cookies/cache with other browser contexts.

Reply:
Done.